src/complete_test.cpp

```diff
@@ -1025,6 +1030,58 @@ int main(int argc, char **argv) {
   sampling_period_accelerometer_client.shutdown();
   time_step_client.call(time_step_srv);
 
+  ////////////////////////////
+  // ALTIMETER METHODS TEST //
+  ////////////////////////////
+
+  ros::ServiceClient set_altimeter_client;
+  webots::set_int altimeter_srv;
+  ros::subscriber sub_altimeter_32;
+
+  set_altimeter_client = n.serviceClinet<webots_ros::set_int>(model_name + "/altimeter/enable");
```

Comment:
Did you run the test? It doesn't look like it would pass as it is added in the robot